services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: str = None,
```

Comment:
Optional[str] here
If default parameter is None it must be Optional
Fix in all places(only your code)